torchgeo/transforms/indices.py

```diff
@@ -147,21 +147,34 @@ def __init__(self, index_red: int, index_nir: int) -> None:
             index_nir: index of the Near Infrared (NIR) band in the image
         """
         super().__init__(index_a=index_red, index_b=index_nir)
-
-
+        
 class AppendNDWI(AppendNormalizedDifferenceIndex):
     """Normalized Difference Water Index (NDWI).
-
     If you use this index in your research, please cite the following paper:
-
     * https://doi.org/10.1080/01431169608948714
     """
 
     def __init__(self, index_green: int, index_nir: int) -> None:
         """Initialize a new transform instance.
-
         Args:
             index_green: index of the Green band in the image
             index_nir: index of the Near Infrared (NIR) band in the image
         """
         super().__init__(index_a=index_green, index_b=index_nir)
+
+class AppendSWI(AppendNormalizedDifferenceIndex):
+    """Normalized Difference Water Index (NDWI).
```

Comment:
Needs to be changed to SWI

Reply:
Changed the name and comments accordingly